res/ui/Tactile/flow.js

```diff
@@ -33,7 +33,7 @@ function create_steps() {
      */
 
     new_scene(Tactile.Start.StartScene);
-    new_step(function(scene) {
+    new_step(function (scene) {
```

Comment:
The rest of the file doesn't use spaces between `function` and`(` (until your editor changed everything) so don't change them all.

```suggestion
    new_step(function(scene) {
```

Reply:
I fix my editor setting, it should be ok now.